README.md

```diff
@@ -1,2 +1,173 @@
+cd bibernate
+```
+#### 2. Build the Bibernate locally by using Maven
+```shell
+mvn clean install
+```
+#### 3. Add the Bibernate dependency into your project `pom.xml`
+```xml
+<dependencies>
+    ...
+    <dependency>
+        <groupId>com.bobocode.blyznytsia</groupId>
+        <artifactId>bibernate</artifactId>
+        <version>${bibernate.version}</version>
+    </dependency>
+    ...
+</dependencies>
+```
+The variable `bibernate.version` is the current version of Bibernate.
+#### 4. Build your application by using maven
+```shell
+mvn clean install
+```
+### Basic usage
+#### Configuration
+To start the project with Bibernate you need to create configuration for your database first.
+1. Create `persistence.yml` file with such structure:
+```yaml
+persistenceUnits:
+  - name: h2
+    dataSource:
+      jdbcUrl: jdbc:h2:mem:testdb
+      username: sa
+      password: password
+      driverClassName: org.h2.Driver
+```
+2. Use the name of persistence unit in `DefaultSessionFactoryBuilder`.
+```java
+SessionFactoryBuilder sessionFactoryBuilder = new DefaultSessionFactoryBuilder("h2");
+SessionFactory sessionFactory = sessionFactoryBuilder.createSessionFactory();
+```
+#### Session usage
+1. You can create the `Session` from `SessionFactory` instance:
+```java
+Session session = sessionFactory.openSession();
+```
+2. To use transaction you can do:
+```java
+Transaction transaction = session.getTransaction();
+transaction.begin();
+// do some action
+transaction.commit();
+// for rollback you can use
+transaction.rollback();
+```
+3. Session has number of method to work with Entity
+- `void persist(Object entity)`:
+
+  This method is used to persist an entity object to the database. The entity object passed as a parameter will be inserted into the database as a new record.
+- `void remove(Object entity)`:
+
+  This method is used to remove an entity object from the database. The entity object passed as a parameter will be deleted from the database.
+- `<T> T find(Class<T> entityClass, Object primaryKey)`:
+
+  This method is used to find an entity in the database by its primary key. The entity class and primary key value are passed as parameters, and the corresponding entity object is returned from the database.
+- `void flush()`:
+  
+  This method is used to synchronize any changes made to the entities in the session with the database. Any pending changes in the session will be written to the database.
+- `void close()`:
+
+  This method is used to close the session and release any resources associated with it, such as database connections. After closing the session, no further operations can be performed on it.
+- `boolean isOpen()`:
+
+  This method is used to check if the session is currently open, i.e., if it has not been closed using the close() method.
+- `Transaction getTransaction()`:
+
+  This method is used to obtain the current transaction associated with the session.
+
+#### Mapping example
```

Comment:
Let's add that mapping is supported only for read operations. And default fetch type is EAGER.